### PR TITLE
BC-1156 add config for ref-thr

### PR DESCRIPTION
**Pull request by BlexToGo**
# Description

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: ansible/host_vars/ref-thr/cfg.yml [hunk 1]
```diff
@@ -1,7 +1,65 @@
...
-ALERT_STATUS_URL: https://status.schulcloud-thueringen.de/
\ No newline at end of file
+ALERT_STATUS_URL: https://status.schulcloud-thueringen.de/
+
+SC_THEME: thr
+SC_TITLE: Thüringer Schulcloud
+SC_SHORT_TITLE: Thüringer Schulcloud
+SC_PRODUCTNAME: Thüringer Schulcloud
+SC_NAV_TITLE: Thüringer Schulcloud
+SC_CONTACT_EMAIL: schulcloud-support@thillm.de
```
Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:12]: Better place where there for the group vars for all the instances
ansible/group_vars/thr/instance_cfg.yml

File: ansible/host_vars/ref-thr/cfg.yml [hunk 2]
```diff
@@ -1,7 +1,65 @@
...
+FEATURE_TEAMS_ENABLED: "true"
+FEATURE_VIDEOCONFERENCE_ENABLED: "false"
+FEATURE_RABBITMQ_ENABLED: "true"
+FEATURE_TSP_AUTO_CONSENT_ENABLED: "true"
+FEATURE_LOG_REQUEST: "false"
+FEATURE_NUXT_SCHOOL_ADMIN_BETA: "true"
+FEATURE_LERNSTORE_ENABLED: "true"
+FEATURE_ES_COLLECTIONS_ENABLED: "true"
+FEATURE_ES_SEARCHABLE_ENABLED: "true"
+FEATURE_ES_MERLIN_ENABLED: "true"
```
Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:41]: Better place where there for the group vars for all the instances
ansible/group_vars/thr/instance_cfg.yml

File: ansible/host_vars/ref-thr/cfg.yml [hunk 3]
```diff
@@ -1,7 +1,65 @@
...
+
+# AUTH
+NOT_AUTHENTICATED_REDIRECT_URL: https://test.schulportal-thueringen.de/cas/login?service=https://test.schulcloud-thueringen.de/tsp-login
+
+# ADMIN TABLES
+ADMIN_TABLES_DISPLAY_CONSENT_COLUMN: "false"
+AWS_REGION: eu-central-1
+STORAGE_SIGNED_URL_EXPIRE: 600
+DISPLAY_REQUEST_LEVEL: -1
+CONTENT_URI: https://content.schulcloud-thueringen.de #Edusharing
```
Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:51]: Better place where there for the group vars for all the instances
ansible/group_vars/thr/instance_cfg.yml

Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:51]: the kommend at the end of the line is bad, posible it's passed to the variable

File: ansible/host_vars/ref-thr/cfg.yml [hunk 4]
```diff
@@ -1,7 +1,65 @@
...
+
+SERVER_LDAP_SYNC_FULL_CRONJOB: "0 */1 * * *"
+JWT_TIMEOUT_SECONDS: 7200
+
+# TSP (Thüringer Schulportal)
+FEATURE_TSP_ENABLED: true
```
Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:57]: Better place where there for the group vars for all the instances
ansible/group_vars/thr/instance_cfg.yml

File: ansible/host_vars/ref-thr/cfg.yml [hunk 5]
```diff
@@ -1,7 +1,65 @@
...
+TSP_API_BASE_URL: https://test.schulportal-thueringen.de
+
+# ANTIVIRUS
+ENABLE_FILE_SECURITY_CHECK: "true"
+FILE_SECURITY_CHECK_SERVICE_URI: http://antivirus-webserver-svc:8080/scan/file
+
+ES_MERLIN_AUTH_URL: http://merlin.nibis.de/auth.php
+LEGACY_COURSE_OVERVIEW_ENABLED: "false"
```
Comment (mamutmk5) [inline ansible/host_vars/ref-thr/cfg.yml:65]: Better place where there for the group vars for all the instances
ansible/group_vars/thr/instance_cfg.yml

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Menny configuration shoud be go from ansible/host_vars/ref-thr/cfg.yml to ansible/group_vars/thr/instance_cfg.yml

missing "SC_FEDERALSTATE: thueringen" what also shoud be go to ansible/group_vars/thr/instance_cfg.yml

File: ansible/group_vars/thr/instance_cfg.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+FEATURE_TSP_AUTO_CONSENT_ENABLED: "true"
+FEATURE_LOG_REQUEST: "false"
+FEATURE_NUXT_SCHOOL_ADMIN_BETA: "true"
+FEATURE_LERNSTORE_ENABLED: "true"
+FEATURE_ES_COLLECTIONS_ENABLED: "true"
+FEATURE_ES_SEARCHABLE_ENABLED: "true"
+FEATURE_ES_MERLIN_ENABLED: "true"
+
+# AUTH
+NOT_AUTHENTICATED_REDIRECT_URL: https://test.schulportal-thueringen.de/cas/login?service=https://test.schulcloud-thueringen.de/tsp-login
```
Comment (mamutmk5) [inline ansible/group_vars/thr/instance_cfg.yml:36]: Also the NOT_AUTHENTICATED_REDIRECT_URL shoud be at ansible/host_vars/ref-thr/cfg.yml

File: ansible/group_vars/thr/instance_cfg.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+STORAGE_SIGNED_URL_EXPIRE: 600
+DISPLAY_REQUEST_LEVEL: -1
+CONTENT_URI: https://content.schulcloud-thueringen.de
+
+SERVER_LDAP_SYNC_FULL_CRONJOB: "0 */1 * * *"
+JWT_TIMEOUT_SECONDS: 7200
+
+# TSP (Thüringer Schulportal)
+FEATURE_TSP_ENABLED: true
+TSP_API_BASE_URL: https://test.schulportal-thueringen.de
```
Comment (mamutmk5) [inline ansible/group_vars/thr/instance_cfg.yml:50]: The URL for the TSP API is expliced instance so it shoud be at ansible/host_vars/ref-thr/cfg.yml

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

File: ansible/group_vars/thr/instance_cfg.yml (hunk 6 above)
Comment (BlexToGo) [reply ansible/group_vars/thr/instance_cfg.yml]: `NOT_AUTHENTICATED_REDIRECT_URL` is now moved to the host config

File: ansible/group_vars/thr/instance_cfg.yml (hunk 7 above)
Comment (BlexToGo) [reply ansible/group_vars/thr/instance_cfg.yml]: `TSP_API_BASE_URL` is now moved to the host config

**Review by mamutmk5 — APPROVED** [review]

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
Please reaktiveade and fix the instance at https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/.github/workflows/rollout.yml#L101-L112

**Comment by BlexToGo** [comment]
> Please reaktiveade and fix the instance at https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/.github/workflows/rollout.yml#L101-L112

@mamutmk5 I have reactived the `ref-thr` snippet in the github rollout and added the three github environment secrets to it, was this what you meant by fixing it?

**Comment by mamutmk5** [comment]
> > Please reaktiveade and fix the instance at https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/.github/workflows/rollout.yml#L101-L112
> 
> @mamutmk5 I have reactived the `ref-thr` snippet in the github rollout and added the three github environment secrets to it, was this what you meant by fixing it?

the code who you add with the commit https://github.com/hpi-schul-cloud/dof_app_deploy/pull/127/commits/2f6172541036203357db707b49368be3ccfff69b

**Review by mamutmk5 — APPROVED** [review]
Please

**Pull request merged**